File: python-notes/12 - conditional_statements (if)/conditional_statements.py
#-----------------------------------------------------------------------------
# Name:        Conditional Statements (conditionalStatements.py)
# Purpose:     To provide information about how conditional statements (if)
#			   work in Python
#
# Author:      Mr. Seidel
# Created:     15-Aug-2018
# Updated:     15-Aug-2018
#-----------------------------------------------------------------------------

# Using 3D coordinates for variable nomenclature
x = 1
y = 2
z = 300

# Basic if statement
if x > 0:
	print('Hello!')

# Using 'elif' in a conditional
if x > 1:
	print("It's not supposed to be!")
elif y == 2:
	print('Yup!')
else:
	print('Nope!')
	
# Using 'and' in a conditional chain
if x > 0 and z > 10:
	print("It's supposed to be here!")
else:
	print("Shouldn't reach here")

# Using 'or' in a conditional chain
if x > 10 or y > 10 or z > 10:
	print('Yup!')
else:
	print('Nope!')

File: javascript-notes/12 - conditionals (if)/conditional_statements.js
/*
  Name:        Conditional Statements (conditional_statements.js)
  Purpose:     To provide information about how conditional statements (if) work in JavaScript

  Author:      Mr. Seidel
  Created:     15-Aug-2018
  Updated:     23-Dec-2022
*/

// Using 3D coordinates for variable nomenclature
const x = 1;
const y = 2;
const z = 300;

// Basic if statement
if (x > 0) {
	console.log("Hello!");
}

// Using "elif" in a conditional
if (x > 1) {
	console.log("It's not supposed to be!")
} else if (y === 2) {
	console.log("Yup!");
} else {
	console.log("Nope!");
}
	
// Using "and" in a conditional chain
if (x > 0 && z > 10) {
	console.log("It's supposed to be here!")
} else {
	console.log("Shouldn't reach here")
}

// Using "or" in a conditional chain
if (x > 10 || y > 10 || z > 10) {
	console.log("Yup!");
} else {
	console.log("Nope!");
}